Makefile

```diff
@@ -169,9 +175,15 @@ build/$(JFRCONV): src/launcher/* build/$(CONVERTER_JAR)
 build/$(LIB_PROFILER): $(SOURCES) $(HEADERS) $(RESOURCES) $(JAVA_HELPER_CLASSES)
 ifeq ($(MERGE),true)
 	for f in src/*.cpp; do echo '#include "'$$f'"'; done |\
-	$(CXX) $(CPPFLAGS) $(CXXFLAGS) $(DEFS) $(INCLUDES) -fPIC -shared -o $@ -xc++ - $(LIBS)
+	$(CXX) $(CPPFLAGS) $(LIB_PROFILER_CXXFLAGS) $(DEFS) $(INCLUDES) -fPIC -shared -o $@ -xc++ - $(LIBS)
 else
-	$(CXX) $(CPPFLAGS) $(CXXFLAGS) $(DEFS) $(INCLUDES) -fPIC -shared -o $@ $(SOURCES) $(LIBS)
+	$(CXX) $(CPPFLAGS) $(LIB_PROFILER_CXXFLAGS) $(DEFS) $(INCLUDES) -fPIC -shared -o $@ $(SOURCES) $(LIBS)
+endif
+
+ifeq ($(OS_TAG),linux)
+	$(STRIP) --only-keep-debug $@ -o $@.debug
```

Comment:
Does `.debug` file get in the release package?

Reply:
yes, e.g. https://github.com/async-profiler/async-profiler/actions/runs/14728842155/artifacts/3028736017

Reply:
It should not. What's the purpose of having a separate .debug file then? The idea was to keep release package small, but provide debuginfo in a separate package.

Reply:
I thought the idea was to keep the .so binary small, will provide in a separate package.